### ISSUE TITLE
ENABLE_NGINX_SSL does not work anymore

### ISSUE THREAD

**Issue by arjanvanderveen** [xirixiz/dsmr-reader-docker#334]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/xirixiz/dsmr-reader-docker/blob/main/.github/SUPPORT.md)

### I've found an issue and checked that ...

- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

When using the environment variable ENABLE_NGINX_SSL set to true, port 443 is not enabled in the Nginx config, due to a faulty sed command.

### Expected behaviour

Nginx is listening on port 443

### Actual behaviour

Nginx is listening on port 80

### Steps to reproduce

Set the environment variable ENABLE_NGINX_SSL=true
Start the docker
The container is listening on port 80 and not port 443

### Docker info

```yaml
n/a
```

### Version

* Docker compose version (type `docker-compose --version`):
* System info (type `uname -a`):
Linux xirixiz-dsmr-reader 4.4.180+ #42962 SMP Sat Apr 8 00:14:23 CST 2023 x86_64 GNU/Linux

### Docker compose

```yaml
n/a
```

### Container logs

n/a

### Additional info

https://github.com/xirixiz/dsmr-reader-docker/commit/d7e77318046baf0da9c151f1423644bc8a1765f0#commitcomment-110488906

When i change the file /etc/cont-init.d/20-set-app-defaults back to the previous code, it works correctly.

**Comment by xirixiz** [comment]
Ja, ik denk dat ik het hele nginx config stuk eruit ga slopen. Het is denk ik beter dat wanneer je een specifieke config wenst, dat je die dan mount in de container. Het geknip en geplak met bash is onbetrouwbaar helaas.

Rolling back.

**Comment by arjanvanderveen** [comment]
Kan het me goed voorstellen. sed is tricky op momenten.

**Comment by xirixiz** [comment]
Afschuwelijk is het hahaha 😄

**Comment by arjanvanderveen** [comment]
Je kan ook nginx includes gebruiken ipv sed. Het is maar een ideetje hoor.
